tests/src/Rules/ThrowsPhpDocRuleTest.php

```diff
@@ -161,4 +168,10 @@ public function testThrowsInGlobalScope(): void
 		$this->analyse(__DIR__ . '/data/throws-in-global-scope.php');
 	}
 
+	public function testMethodWhitelist(): void
+	{
+		$this->methodWhitelist = [TestCase::class => '/^test/'];
```

Comment:
@marcospassos Inconsistent structure. If I follow readme it should be `[TestCase::class => ['/^test/']]`

Reply:
My fault, it's supposed to be a map of class names and patterns as key-value pairs. I don't see a significant advantage in supporting multiples patterns per class, and it brings performance penalties. It's better to combine multiples patterns into a single one.

Reply:
@marcospassos agree with map